src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.card__stars {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  li {
```

Comment:
Don't use tag names for styling (except html and body)
<img width="867" alt="image" src="https://github.com/user-attachments/assets/ddf10a3f-456c-43ef-964f-b844036ce7a3">